test/Directory.Build.targets

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(IsUnitTestProject)' == 'true'">
-    <ProjectReference Include="$(MSBuildThisFileDirectory)\TestUtilities\TestUtilities.csproj" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)\TestUtilities\TestUtilities.csproj" NoWarn="RT0002" />
```

Comment:
Please add a comment on what this is for.

Reply:
I removed this sequence and instead adding explicit project references to the projects that needed it.